1-exercises/B-while-loop/exercise.js

```diff
@@ -6,9 +6,16 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  let i = 0;
+  let evens = [];
+    
+  while (evens.length < n) {
```

Comment:
Good job making the array of even numbers. One minor point is that the instructions were saying you should "log (using console.log) the first n even numbers as a comma-seperated string".
At the moment you are simply returning the array.